src/ng-select/items-list.ts

```diff
@@ -41,16 +43,16 @@ export class ItemsList {
         item.selected = true;
     }
 
-    findItem(value, bindValue: string, bindLabel: string): NgOption {
+    findItem(value, bindValue: string): NgOption {
         if (!value) {
             return null;
         }
         if (bindValue) {
-            return this.items.find(x => x[bindValue] === value);
+            return this.items.find(item => item.value[bindValue] === value);
```

Comment:
How will this work if bindValue would be nested? Item.value[“company.name”] will always return undefined